src/containers/AdminCampaignStats/index.jsx

```diff
@@ -253,13 +254,22 @@ class AdminCampaignStats extends React.Component {
       ? DateTime.fromISO(campaign.dueBy).toFormat("DD")
       : "No Due Date";
     const isOverdue = DateTime.local() >= DateTime.fromISO(campaign.dueBy);
-    const newTitle = `${this.props.organizationData.organization.name} - Campaigns - ${campaignId}: ${campaign.title}`;
+    const newTitle = `${organization.name} - Campaigns - ${campaignId}: ${campaign.title}`;
 
     // only a superadmin with multiple active orgs can copy a campaign to another org
     const orgLength = this.props.orgs?.organizations?.length;
     const onlyCopyCampaignSameOrg =
       !isSuperadmin || orgLength === undefined || orgLength < 2;
 
+    let msgServiceName = messagingService?.name;
```

Comment:
nit: prefer ternary operator
nit: prefer full variable name, we have `showMessagingServiceName`, can use `messagingServiceName`

Reply:
Can we discuss the 2nd one? I actually think shortening common things like organization -> org or messaging -> msg for a variable name is fine and doesn't decrease readability

Reply:
> Can we discuss the 2nd one? I actually think shortening common things like organization -> org or messaging -> msg for a variable name is fine and doesn't decrease readability

It doesn't, that's why it's categorized as a nit instead of suggestion :)

Reply:
fair 😆